store/postgres/src/entities.rs

```diff
@@ -51,6 +51,11 @@ use crate::filter::{build_filter, store_filter};
 use crate::functions::set_config;
 use crate::jsonb::PgJsonbExpressionMethods as _;
 
+/// The size of string prefixes that we index. This should be large enough
+/// that we catch most strings, but small enough so that we can still insert
+/// it into a Postgres BTree index
+pub(crate) const STRING_PREFIX_SIZE: usize = 2048;
```

Comment:
If we're not sure, should we make this a `GRAPH_STORE_POSTGRES_STRING_INDEX_PREFIX_SIZE` (or something else that is prefixed with GRAPH_) environment variable?

Reply:
Ok